test/test_generate_pedigree.py

```diff
@@ -0,0 +1,159 @@
+"""
+methods for testing the sample-metadata API queries
+"""
+
+from copy import deepcopy
+
+from unittest.mock import patch
+
+import json
+import os
+
+from helpers.pedigree_from_sample_metadata import (
+    ext_to_int_sample_map,
+    get_fat_pedigree,
+    get_pedigree_for_project,
+)
+
+
+PWD = os.path.dirname(__file__)
+PROJECT = 'fake-project'
+INPUT = os.path.join(PWD, 'input')
+JSON_PED = os.path.join(INPUT, 'mock_pedigree.json')
+LOOKUP_PED = os.path.join(INPUT, 'mock_sm_lookup.json')
+
+SAMPLE_TO_CPG = {
+    'sam1': ['cpg1'],
+    'sam2': ['cpg2'],
+    'sam3': ['cpg3'],
+}
+DIRTY_PED = [
+    {
+        'family_id': 'FAM1',
+        'individual_id': 'sam1',
+        'paternal_id': 'sam2',
+        'maternal_id': 'sam3',
+        'sex': 1,
+        'affected': 1,
+    },
+    {
+        'family_id': 'FAMX',
+        'individual_id': 'samx',
+        'paternal_id': 'samy',
+        'maternal_id': 'samz',
+        'sex': 1,
+        'affected': 1,
+    },
+]
+
+
+@patch('helpers.pedigree_from_sample_metadata.FamilyApi.get_pedigree')
+def test_get_pedigree_for_project(
+    get_ped_patch,
+):
+    """
+
+    :param get_ped_patch:
+    :return:
+    """
+
+    with open(JSON_PED, 'r', encoding='utf-8') as handle:
+        payload = json.load(handle)
+        get_ped_patch.return_value = payload
+        result = get_pedigree_for_project(project=PROJECT)
+        assert result == payload
```

Comment:
I don't think this is a useful test, the inner function only makes one call that's patched